libcloudforensics/providers/gcp/internal/storage.py

```diff
@@ -131,6 +131,21 @@ def GetBucketACLs(self,
         ret[item['role']].append(member)
     return ret
 
+  def ListBuckets(self) -> List[Dict[str, Any]]:
+    """List buckets in a Google Cloud project.
+
+    Args:
```

Comment:
This could probably be removed all together?

Reply:
Done.